<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Av1an_main_windows</class>
 <widget class="QMainWindow" name="Av1an_main_windows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>200</horstretch>
    <verstretch>200</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>200</horstretch>
     <verstretch>200</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
      <item>
       <layout class="QHBoxLayout" name="ButtonSelect" stretch="1,0,1,0">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="Input_Button">
          <property name="text">
           <string>Input / Output</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Video_Settings_Button">
          <property name="text">
           <string>Encoding Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="MainWidget">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>400</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="visible">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0" colspan="5">
             <widget class="QLabel" name="DropInLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>100</horstretch>
                <verstretch>100</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>36</pointsize>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">            QLabel{

                border: 4px dashed #aaa

            }</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string>Drop In File Or Click For Import</string>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Select Output:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="pushButtonSelectOutput">
              <property name="text">
               <string>Select...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageAv1AnSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="2">
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string>Encoder</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="7" column="0">
                 <widget class="QLabel" name="labelEncoder_5">
                  <property name="text">
                   <string>Passes:</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0" colspan="2">
                 <widget class="QLabel" name="labelBitDepth_3">
                  <property name="text">
                   <string>Bit-Depth:</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="2">
                 <widget class="QComboBox" name="comboBoxBitDepth">
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>12</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QLabel" name="labelEncoder_6">
                  <property name="text">
                   <string>Encoder:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QComboBox" name="comboBoxEncoder">
                  <item>
                   <property name="text">
                    <string>aom</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>rav1e</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>svt-av1</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="labelEncoder_7">
                  <property name="text">
                   <string>Speed:</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="2">
                 <widget class="QComboBox" name="comboBoxPasses">
                  <item>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="labelSpeed">
                  <property name="text">
                   <string>4</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QSlider" name="horizontalSliderSpeed">
                  <property name="maximum">
                   <number>9</number>
                  </property>
                  <property name="value">
                   <number>6</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::TicksBelow</enum>
                  </property>
                  <property name="tickInterval">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Segmenting</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="3" column="0">
                 <widget class="QCheckBox" name="checkBox">
                  <property name="text">
                   <string>Extra Splits:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelSplitting">
                  <property name="text">
                   <string>Splitting:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Workers:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="comboBoxAv1anSplitting">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Method used for generating splits. (Default: PySceneDetect)&lt;/p&gt;&lt;p&gt;Options:  &lt;/p&gt;&lt;p&gt;PyScenedetect, content based scenedetection with threshold.&lt;/p&gt;&lt;p&gt;aom_keyframes, using stat file of 1 pass of aomenc encode&lt;/p&gt;&lt;p&gt;to get exact place where encoder will place new keyframes.&lt;/p&gt;&lt;p&gt; (Keep in mind that speed also depends on set aomenc parameters)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>PySceneDetect</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>aom_keyframes</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="spinBoxAv1anWorkers">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>256</number>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QSpinBox" name="spinBoxAv1anExtraSplits">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;pre style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Mono';&quot;&gt;Adding extra splits if frame distance beetween splits bigger than&lt;/span&gt;&lt;/pre&gt;&lt;pre style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Mono';&quot;&gt;given value. Works with/without PySceneDetect&lt;/span&gt;&lt;/pre&gt;&lt;pre style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Mono';&quot;&gt;Example: 1000 frames video with single scene,&lt;/span&gt;&lt;/pre&gt;&lt;pre style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Mono';&quot;&gt;-xs 200 will add splits at 200,400,600,800.&lt;/span&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>10000</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelPySceneThreshold">
                  <property name="text">
                   <string>Threshold:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="spinBoxPySceneThreshold">
                  <property name="toolTip">
                   <string>PySceneDetect threshold for scene detection Default: 35</string>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="value">
                   <number>35</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QGroupBox" name="groupBox_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Target Quality (VMAF)</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="1" column="0">
                 <widget class="QCheckBox" name="checkBoxMinQualityVMAF">
                  <property name="text">
                   <string>Min Quality:</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QComboBox" name="comboBoxVMAFResolution">
                  <item>
                   <property name="text">
                    <string>1920x1080</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>3840x2160</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>VMAF Probes:</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QSpinBox" name="spinBoxVMAFProbes">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of probes for interpolation.&lt;/p&gt;&lt;p&gt;1 and 2 probes have special cases to try to work with few data points.&lt;/p&gt;&lt;p&gt;Optimal is 4-6 probes. Default: 4&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>10</number>
                  </property>
                  <property name="value">
                   <number>4</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="spinBoxMaxQVMAF">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimum Quality (max quantizer) value limit for Target VMAF&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="spinBoxTargetVMAF">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;pre style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Mono';&quot;&gt;Vmaf value to target. Supported for all encoders(Exception:VVC).&lt;/span&gt;&lt;/pre&gt;&lt;pre style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Mono';&quot;&gt;Works best in range 85-97.&lt;/span&gt;&lt;/pre&gt;&lt;pre style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Mono';&quot;&gt;When using this mode specify full encoding options.&lt;/span&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="minimum">
                   <number>50</number>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="value">
                   <number>95</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QCheckBox" name="checkBoxMaxQualityVMAF">
                  <property name="text">
                   <string>Max Quality:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spinBoxMinQVMAF">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum Quality (min quantizer) value limit for Target VMAF&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>VMAF Resolution:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Target Quality (VMAF):</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Audio</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Codec:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBox">
                  <item>
                   <property name="text">
                    <string>Opus</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>AC3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>AAC</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>MP3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Copy Audio</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Bitrate (KBit/s):</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spinBox">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>640</number>
                  </property>
                  <property name="value">
                   <number>128</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Channels:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="comboBox_2">
                  <property name="currentIndex">
                   <number>1</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>1.0 (Mono)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2.0 (Stereo)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>5.1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>7.1</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Progress_Bar" stretch="1,0,0">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>100</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="StartStop">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
